src/app/pages/authentication/successful-signup/successful-signup.page.html

```diff
@@ -21,7 +21,7 @@ <h2>Hello,
 
   <ion-row>
     <ion-col>
-      <ion-button type="submit" expand="block" shape="round" padding (click)="submit()">Let's get rollin'</ion-button>
+      <ion-button type="submit" expand="block" shape="round" class="padding" (click)="submit()">Let's get rollin'</ion-button>
```

Comment:
Not `ion-padding`?

Reply:
Hadn't caught that one because I hadn't navigated to the page to see the warning. Nice catch.